rna/rna-bam-sort-and-index/rna-index.wdl

```diff
@@ -88,6 +88,7 @@ task samtools_index {
     echo "$(date "+%Y-%m-%d %H:%M:%S") Starting samtools flagstat on new bam"
     samtools flagstat "${sample_name}.samtools.bam" > flagstat.new.txt
 
+    # Verifying the old and new BAMs produce the same flagstats
```

Comment:
Ah - to be consistent with the rest of the file:

echo "$(date "+%Y-%m-%d %H:%M:%S") Verifying the old and new BAMs produce the same flagstats"